pcc/src/lib/addonApiHelper.ts

```diff
@@ -0,0 +1,74 @@
+import axios, { HttpStatusCode } from 'axios';
+import { getLocalAuthDetails } from './localStorage';
+import { HTTPNotFound } from '../cli/exceptions';
+
+const API_KEY_ENDPOINT =
+  'https://us-central1-pantheon-content-cloud-staging.cloudfunctions.net/addOnApi/api-key';
+const OAUTH_ENDPOINT =
+  'https://us-central1-pantheon-content-cloud-staging.cloudfunctions.net/addOnApi/oauth';
```

Comment:
1.) These need to point to prod by default.
2.) There needs to be a way for us to change to staging for development purposes.

Reply:
Oh yes, I missed this. Good catch.